raphtory/src/algorithms/motifs/local_temporal_three_node_motifs.rs

```diff
@@ -148,7 +148,7 @@ where
             .iter()
             .flat_map(|e| e.explode())
             .merge_by(inc.iter().flat_map(|e| e.explode()), |e1, e2| {
-                e1.time_and_index() < e2.time_and_index()
+                e1.time_and_index().unwrap() < e2.time_and_index().unwrap()
```

Comment:
@miratepuffin Make a ticket for exploded edges being converted into different types